### PR TITLE
ovs-4139-initial-etcd-value-update

### PR DESCRIPTION
**Pull request by pploegaert**

### CODE REVIEW

File: ovs/lib/albacontroller.py [hunk 1]
```diff
@@ -251,7 +251,10 @@ def add_cluster(alba_backend_guid):
 
         etcd_key = AlbaController.ETCD_NR_OF_AGENTS_KEY.format(alba_backend_guid)
         nr_of_storage_nodes = len(AlbaNodeList.get_albanodes())
-        EtcdConfiguration.set(etcd_key, nr_of_storage_nodes)
```
Comment (kvanhijf) [inline ovs/lib/albacontroller.py]: We should not manually set the amount of storage nodes to 1 if none are present
The core problem here has most likely something to do with the order in which our hooks are executed during ovs setup

OVS setup used to model the ASD nodes right away, but since the introduction of the ASDmanager setup, this isn't done right away anymore. Something to investigate and revert this change

Comment (khenderick) [inline ovs/lib/albacontroller.py]: The setup still does that, but at this point, at the node being installed, the asd-manager setup has not been executed yet. So basically, the correct thing to do here is make sure that the AlbaController.add_cluster also (at the end) runs that one method that does the node detection.
